src/v/raft/tests/basic_raft_fixture_test.cc

```diff
@@ -1050,3 +1050,42 @@ TEST_F_CORO(raft_fixture, test_replicate_abort_source) {
     ASSERT_TRUE_CORO(r_2.has_error());
     ASSERT_EQ_CORO(r_2.error(), raft::errc::replicate_first_stage_exception);
 }
+
+TEST_F_CORO(raft_fixture, test_leadership_blocked_replicas_can_elect_leader) {
+    co_await create_simple_group(3);
+    auto leader_id = co_await wait_for_leader(10s);
+    auto& leader_node = node(leader_id);
+    // replicate some batches to all replicas
+    auto res = co_await leader_node.raft()->replicate(
+      make_batches(1, 1, 128),
+      replicate_options(consistency_level::quorum_ack));
+    ASSERT_FALSE_CORO(res.has_error());
+    auto follower = random_follower_id().value();
+    /**
+     * Block append entries to one of the followers
+     */
+    leader_node.on_dispatch([follower](model::node_id node, raft::msg_type mt) {
+        if (mt == raft::msg_type::append_entries && follower == node) {
+            throw std::runtime_error("error");
+        }
+        return ss::now();
+    });
+    // replicate more batches.
+    res = co_await leader_node.raft()->replicate(
+      make_batches(1, 1, 128),
+      replicate_options(consistency_level::quorum_ack));
+    ASSERT_FALSE_CORO(res.has_error());
+    /**
+     * The only node that can be elected is the one with shortest log, we have
+     * the other two with blocked leadership
+     */
```

Comment:
This comment confused me at first, as it describes the situation at step-down, which happens further down. 
1) Could you move it one line down so that the blank line does not separate it from the blocking loop it annotates?
2) Could you rephrase it in imperative mood, e.g. "block leadership on two nodes so that the only node can be elected ..."? Or, alternatively, move the comment below the blocking loop so that we have them with blocked leadership for real.